packages/api-reference/src/components/Content/ClientLibraries/ClientSelector.vue

```diff
@@ -1,89 +1,50 @@
 <script setup lang="ts">
-import { useResizeObserver } from '@vueuse/core'
+import { ScalarIcon } from '@scalar/components'
 import { type TargetId } from 'httpsnippet-lite'
-import { computed, ref } from 'vue'
+import { ref } from 'vue'
 
 import { useSnippetTargets } from '../../../hooks'
 import { type SelectedClient, useTemplateStore } from '../../../stores/template'
-import { Icon } from '../../Icon'
 
 // Use the template store to keep it accessible globally
 const { state, setItem, getClientTitle, getTargetTitle } = useTemplateStore()
-
 const { availableTargets } = useSnippetTargets()
-
-const isSmall = ref(false)
 const containerRef = ref<HTMLElement>()
 
-useResizeObserver(
-  containerRef,
-  (entries) => (isSmall.value = entries[0].contentRect.width < 500),
-)
-
 // Show popular clients with an icon, not just in a select.
-const featuredClients = computed<SelectedClient[]>(() =>
-  isSmall.value
-    ? // Mobile
-      [
-        {
-          targetKey: 'shell',
-          clientKey: 'curl',
-        },
-        {
-          targetKey: 'ruby',
-          clientKey: 'native',
-        },
-        {
-          targetKey: 'node',
-          clientKey: 'undici',
-        },
-        {
-          targetKey: 'python',
-          clientKey: 'python3',
-        },
-      ]
-    : // Desktop
-      [
-        {
-          targetKey: 'shell',
-          clientKey: 'curl',
-        },
-        {
-          targetKey: 'ruby',
-          clientKey: 'native',
-        },
-        {
-          targetKey: 'node',
-          clientKey: 'undici',
-        },
-        {
-          targetKey: 'php',
-          clientKey: 'guzzle',
-        },
-        {
-          targetKey: 'python',
-          clientKey: 'python3',
-        },
-        {
-          targetKey: 'c',
-          clientKey: 'libcurl',
-        },
-      ],
-)
+const featuredClients = [
```

Comment:
If we want to keep the smaller set on mobile, we can easily do so using css. Just left it out for now because it doesn't look half bad

Reply:
I think this overcrowds it on the classic layout 
<img width="490" alt="Google Chrome-2024-03-21-10-23-10@2x" src="https://github.com/scalar/scalar/assets/6374090/1de04739-1ed9-47c4-8437-519adee05bb2">

Reply:
definitely, I always forget about classic haha. Will add some css

Reply:
Done!